API/Audio/AudioAPI.cpp

```diff
@@ -1,3 +1,86 @@
 //
 // Created by Storm on 05/10/2020.
 //
```

Comment:
Misschien moeten we even bespreken of we header files in onze repo willen. Het is nu in elk geval niet consistent.